kevm-pyk/src/kevm_pyk/kproj/evm-semantics/driver.md

```diff
@@ -73,6 +73,7 @@ To do so, we'll extend sort `JSON` with some EVM specific syntax, and provide a
     rule <k> startTx => loadTx( #sender( #getTxData(TXID), TW, TR, TS, B ) ) ... </k>
          <txPending> ListItem(TXID:Int) ... </txPending>
          <chainID> B </chainID>
+         <gas> _ => 0 </gas>
```

Comment:
Initially, I tried using `clearTX` after `finalizeTx(_)`; however, this increased the number of failed tests.